stable/application-mesh/tests/golden/components/envoy-filter-max-body-size.values.yaml

```diff
@@ -0,0 +1,14 @@
+global:
+  application:
+    product: leads-broker
+    environment: test
+    version: "0.0.0"
+
+destinations:
+  - component: webserver
+
+ingress:
+  - host: leads-broker-st1.matic.com
+    gateway: external-matic-com
+    destination: webserver
+    max_request_bytes: "67108864"
```

Comment:
Why is this value a string? Is it required? What happens if I don't specify quotes

Reply:
In this case, you should transform it to int representation when rendering. I assure you, next time someone edits this setting - they will forget to add quotes